src/pudl/transform/eia923.py

```diff
@@ -925,6 +930,8 @@ def generation(eia923_dfs, eia923_transformed_dfs):
         .pipe(_yearly_to_monthly_records)
         .pipe(pudl.helpers.fix_eia_na)
         .pipe(pudl.helpers.convert_to_date)
+        .rename(columns={"eia_sector": "sector_id_eia"})
+        .rename(columns={"sector_name": "sector_name_eia"})
```

Comment:
As above, do these columns exist / need to be renamed now?

Reply:
Ugh. These changes were unintentional, of course. I had removed them, but git wanted me to pull in changes before I committed (which obviously was a red flag), and they must have come back then.